app/imports/api/users/server/publications.js

```diff
@@ -52,3 +52,16 @@ Meteor.publish('extendedUser', function publishExtendedUser() {
         },
     });
 });
+
+Meteor.publish('githubUsername', function publishExtendedUser() {
```

Comment:
`publishExtendedUser` this function name is used by other publication. If one of them throw an error you won't know which one, change this name.